tasks/git/src/main/java/com/walmartlabs/concord/plugins/git/Utils.java

```diff
@@ -52,6 +56,23 @@ public static String getUrl(Map<String, Object> defaults, Map<String, Object> in
         return v;
     }
 
+    public static String hideSensitiveData(String s, Secret secret) {
+        if (s == null) {
+            return null;
+        }
+
+        if (secret instanceof UsernamePassword) {
+            char[] password = ((UsernamePassword) secret).getPassword();
+            if (password != null) {
+                s = s.replaceAll(Pattern.quote(new String(password)), "***");
```

Comment:
Does it have to be a regex? Can't we just do `s.replace` instead of `replaceAll`?

Reply:
yeah that should be sufficient.